irohad/torii/impl/command_service_transport_grpc.cpp

```diff
@@ -56,6 +57,8 @@ namespace iroha {
         const iroha::protocol::Transaction *request,
         google::protobuf::Empty *response) {
       iroha::protocol::TxList single_tx_list;
+      shared_model::proto::Transaction tx(*request);
+      log_->trace("Received tx {} at endpoint", tx.hash());
```

Comment:
please move this logging to `CommandServiceTransportGrpc::ListTorii` after
```c++
auto transactions = deserializeTransactions(request);
```
so that we 
- do not have to parse every tx twice
- also log ListTorii txs